shared/modules/consensus_module.go

```diff
@@ -74,6 +73,8 @@ type ConsensusPacemaker interface {
 type ConsensusStateSync interface {
 	GetNodeIdFromNodeAddress(string) (uint64, error)
 	GetNodeAddress() string
+	IsSynched() (bool, error)
+	GetValidatorsAtHeight(uint64) ([]*types.Actor, error)
```

Comment:
Why are we exposing this? I couldn't find any usages outside of the `consensus` package.

General piece of feedback: I know it's really easy to add this in the shared interface, but always think twice if its necessary.

Reply:
Very good point, thanks. Removed.